clio-util/src/main/scala/org/broadinstitute/clio/util/json/ModelAutoDerivation.scala

```diff
@@ -41,7 +41,7 @@ trait ModelAutoDerivation extends AutoDerivation {
     * By default, compact JSON as much as possible by removing spaces and keys with null / None values.
     */
   implicit val defaultPrinter: Printer =
-    Printer.noSpaces.copy(dropNullValues = true)
+    Printer.noSpaces.copy(dropNullValues = true, reuseWriters = true)
```

Comment:
Stumbled across this optimization option. According to [this PR](https://github.com/circe/circe/pull/657) it has a significant affect on throughput / allocations, but it's not enabled by default because of some potential problem with Java's `ThreadLocal`... not sure how to tell if we're at risk.

Reply:
This makes me nervous unless we know we'll notice any corruption of the output.
(Es will probably notice bad JSON, but maybe not missing fields or objects.)